docs/samples/metrics-and-monitoring/prometheus/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: prometheus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: prometheus
+subjects:
+- kind: ServiceAccount
+  name: prometheus
+  namespace: default
```

Comment:
Do you mean `kfserving-monitoring` over here? Maybe we should parameterize it so users can also use it on the istio-system prometheus.

Reply:
The `namespace` field in that line is simply a place holder and its value will be overwritten by the top-level kustomization. Nevertheless, the `default` value might be confusing, so I replaced it with `kfserving-monitoring` in my latest commit.

I also added explicit instructions for how to kustomize the install namespace.